Dockerfile-fix

codesimo opened the pull request:
Il vecchio dockerfile non funzionava, l'ho aggiornato usando ubuntu 20.04 come base e applicando tutti i fix descritti nel readme.
Inoltre credo che il file token.conf non venga più usato.

Helias commented on Dockerfile:
```diff
@@ -1,23 +1,37 @@
 ENV DMI_BOT_REPO    https://github.com/UNICT-DMI/Telegram-DMI-Bot.git
 ENV DMI_BOT_DIR    /usr/local
 
+#Fix to install software-properties-common, needed to use add-apt-repository
```
```suggestion
# fix to install software-properties-common, needed to use add-apt-repository
```

Helias commented on Dockerfile:
```diff
@@ -1,23 +1,37 @@
+	language-pack-it\
+	software-properties-common
+
+#Fix libqtwebkit-dev missing
```
```suggestion
# fix libqtwebkit-dev missing
```

Helias commented on Dockerfile:
```diff
@@ -1,23 +1,37 @@
+#Fix libqtwebkit-dev missing
+RUN add-apt-repository ppa:rock-core/qt4 -y && apt-get install -y libqtwebkit-dev
+
+#Fix webkit-server
```
```suggestion
# fix webkit-server
```

Helias commented on Dockerfile:
```diff
@@ -1,23 +1,37 @@
 RUN mkdir -p $DMI_BOT_DIR && \
-  cd $DMI_BOT_DIR && \
-  git clone -b master $DMI_BOT_REPO dmibot
+	cd $DMI_BOT_DIR && \
+	git clone -b master $DMI_BOT_REPO dmibot
 
 RUN pip3 install -r $DMI_BOT_DIR/dmibot/requirements.txt
```
Too much spaces here, be coherent with the rest

Helias commented on Dockerfile:
```diff
@@ -1,23 +1,37 @@
 RUN cp $DMI_BOT_DIR/dmibot/data/DMI_DB.db.dist $DMI_BOT_DIR/dmibot/data/DMI_DB.db
 RUN cp $DMI_BOT_DIR/dmibot/config/settings.yaml.dist $DMI_BOT_DIR/dmibot/config/settings.yaml
-RUN echo $TOKEN > $DMI_BOT_DIR/dmibot/config/token.conf
+#RUN echo $TOKEN > $DMI_BOT_DIR/dmibot/config/token.conf
```
useless comment, remove it
```suggestion
```

codesimo commented:
I removed dryscrape and replaced it with requests in local, but TendTo did the same creating a PR. He added some logging stuff and changed how the dockerfile works, so this PR becomes useless.
Closing
